cmd/xl-storage-disk-id-check.go

```diff
@@ -81,11 +81,14 @@ type xlStorageDiskIDCheck struct {
 	totalErrsAvailability uint64 // Captures all data availability errors such as permission denied, faulty disk and timeout errors.
 	totalErrsTimeout      uint64 // Captures all timeout only errors
 	// apiCalls should be placed first so alignment is guaranteed for atomic operations.
-	apiCalls     [storageMetricLast]uint64
-	apiLatencies [storageMetricLast]*lockedLastMinuteLatency
-	diskID       string
-	storage      *xlStorage
-	health       *diskHealthTracker
+	apiCalls          [storageMetricLast]uint64
+	apiLatencies      [storageMetricLast]*lockedLastMinuteLatency
+	diskID            string
+	storage           *xlStorage
+	health            *diskHealthTracker
+	diskStartChecking int
+	diskMaxConcurrent int
```

Comment:
Let's add comments for those variables; I understand diskMaxConcurrent, but not diskStartChecking